.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
What's driving this?

Reply:
Nothing really other than we can support it. I could remove it from this PR if necessary